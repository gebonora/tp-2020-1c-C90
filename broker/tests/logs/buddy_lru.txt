Partición 1: | 0 - 31 | [X] | Size: 32 b | LRU: 11 | COLA: CATCH | ID: 6 |
Partición 2: | 32 - 63 | [X] | Size: 32 b | LRU: 10 | COLA: NEW | ID: 3 |
