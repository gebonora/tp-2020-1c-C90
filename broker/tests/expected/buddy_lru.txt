Partición 1: | 0 - 31 | [X] | Size: 32 b | LRU: 6 | COLA: CATCH | ID: 5 |
Partición 2: | 32 - 63 | [X] | Size: 32 b | LRU: 5 | COLA: NEW | ID: 3 |
